wrouter/routerimpl.go

```diff
@@ -43,4 +43,6 @@ type RouterImpl interface {
 	// the number of parameters in the path. The variable names of the parameters in the path are provided in order
 	// in the "pathVarNames" slice.
 	PathParams(req *http.Request, pathVarNames []string) map[string]string
+
+	RegisterNotFoundHandler(handler http.Handler)
```

Comment:
This is technically a break in the case that someone implemented a third router type, but that's pretty unlikely right?